### PR TITLE
Add helper view for creating user and pairing

### PR DESCRIPTION
**Pull request by tpazderka**

### CODE REVIEW

File: django_mojeid_connect/templates/create_user.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block title %}Create user{% endblock %}
+{% block content %}
+    Create a new user using the form below.
+    You will be redirected to mojeID login and your user will be bound to the mojeID account for future logins.
+    <form action="" method="post">
+        {% csrf_token %}
+        {{ form.as_p}}
```
Comment (ziima) [inline django_mojeid_connect/templates/create_user.html]: Missing space before closing bracket.

File: django_mojeid_connect/views.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+"""Views for django_mojeid_connect."""
+from __future__ import unicode_literals
+
+from django.contrib.auth import get_user_model, login
+from django.urls import reverse_lazy
+from django.views.generic import CreateView
+
+
+class CreateUser(CreateView):
+    """View for creating and pairing user to mojeID."""
```
Comment (ziima) [inline django_mojeid_connect/views.py]: It would be better to describe it in more details.

**Review by ziima — CHANGES_REQUESTED** [review]

**Comment by tpazderka** [comment]
Fixed.

File: django_mojeid_connect/views.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+from django.urls import reverse_lazy
+from django.views.generic import CreateView
+
+
+class CreateUser(CreateView):
+    """
+    View for creating and pairing user to mojeID.
+
+    This is intended as a demonstration view for pairing of an existing user.
+    If user is logged in, then the only requested attribute is `sub` which is then used to vreate the binding.
```
Comment (ziima) [inline django_mojeid_connect/views.py]: `... to create ...`

**Review by ziima — CHANGES_REQUESTED** [review]

File: django_mojeid_connect/views.py (hunk 3 above)
Comment (tpazderka) [reply django_mojeid_connect/views.py]: Fixed.

**Review by ziima — APPROVED** [review]

**Pull request merged**